components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -452,8 +452,10 @@ abstract class EngineSession(
 
     /**
      * Reloads the current URL.
+     *
+     * @param flags the [LoadUrlFlags] to use when loading the provider url.
```

Comment:
nit: when reloading the current URL.

nit nit: There's also a type in (probably my) original comment. provided URL not, provider URL.

Reply:
Fixed